src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj

```diff
@@ -49,6 +50,8 @@
     <Compile Include="Internal\SpscTargetCore.cs" />
     <Compile Include="Internal\TargetCore.cs" />
     <Compile Include="Internal\TargetRegistry.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.0' OR '$(TargetGroup)' == 'netstandard1.1'">
```

Comment:
I guess what I'm asking is... why?  Why have `#if !THREADING` rather than `#if USE_INTERNAL_THREADING`, and then have all of these conditions match, i.e. definining USE_INTERNAL_THREADING and including the relevant files to use internal threading when you want to, well, use internal threading?